classes/core/PKPRequest.inc.php

```diff
@@ -815,10 +815,11 @@ function url($context = null, $page = null, $op = null, $path = null,
 	 *
 	 * @return PKPRequest
 	 */
-	function &_checkThis() {
-		if (isset($this) && is_a($this, 'PKPRequest')) {
+	static function &_checkThis() {
+		// Probably not a good fix, but this is working as expected.
+/*		if (isset($this) && is_a($this, 'PKPRequest')) {
 			return $this;
-		} else {
+		} else {*/
```

Comment:
(This stop-gap is necessary while we refactor `Request` static calls into instance calls.)